semantics/cpp14/language/common/typing.k

```diff
@@ -448,6 +474,12 @@ module CPP-TYPING
      rule min(T:CPPWideCharType) => min(underlyingType(T))
      rule max(T:CPPWideCharType) => max(underlyingType(T))
 
+     rule min(t(_, _, scopedEnum(_, UT::CPPIntegerType))) => min(UT)
+     rule max(t(_, _, scopedEnum(_, UT::CPPIntegerType))) => max(UT)
+
+     rule min(t(_, _, unscopedEnum(E::EnumId, _, _))) => #getEnumMin(#getEnumInfo(E, #configuration))
```

Comment:
this doesn't seem quite right. In the case of an unscoped enum whose type is fixed, it should have the same values as the scoped enum above.